### ISSUE TITLE
Sum list vs generator comprehesion detection with startpoint arg

### ISSUE THREAD

**Issue by wanaryytel** [adamchainz/flake8-comprehensions#178]
So right now the linter highlights this:
```sum([x + 1 for x in range(30)])``` and for all the right reasons.
However, this isn't detected right now:
```sum([x + 1 for x in range(30)], 0)```

**Comment by adamchainz** [comment]
Fixed in version 2.3.0: https://pypi.org/project/flake8-comprehensions/2.3.0/

Thanks!
